packages/sources/src/components/responsive-element/responsive-element.stories-of.ts

```diff
@@ -9,8 +10,13 @@ const demoGrid = [
   ['col-xs-3', 'col-xs-3', 'col-xs-3', 'col-xs-3']
 ];
 
+type ResponsiveElementTemplateFnType<TemplateFnReturnType> = (
+  onSizeChange: (event: CustomEvent<string>) => void,
+  children: string[][],
```

Comment:
is dit niet `TemplateFnReturnType`

Reply:
Nee, daar komt die `const demoGrid` die erboven staat in. in `responsive-element.stories.ts` wordt daar een template van gemaakt. Kan mogelijk omgevormd worden tot `TemplateFnReturnType`, maar dan moet het een en ander worden omgeschreven.